iOS35/Common/SnappingCollectionViewFlowLayout.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  SnappingCollectionViewFlowLayout.swift
+//  iOS35
+//
+//  Created by 우상욱 on 10/31/24.
+//
+
+import UIKit
+
+class SnappingCollectionViewFlowLayout: UICollectionViewFlowLayout {
+    override func targetContentOffset(forProposedContentOffset proposedContentOffset: CGPoint, withScrollingVelocity velocity: CGPoint) -> CGPoint {
+           guard let collectionView = collectionView else { return super.targetContentOffset(forProposedContentOffset: proposedContentOffset, withScrollingVelocity: velocity) }
+
+           var offsetAdjustment = CGFloat.greatestFiniteMagnitude
+           let horizontalOffset = proposedContentOffset.x + collectionView.contentInset.left
+
+           let targetRect = CGRect(x: proposedContentOffset.x, y: 0, width: collectionView.bounds.size.width, height: collectionView.bounds.size.height)
+
+           let layoutAttributesArray = super.layoutAttributesForElements(in: targetRect)
+
+           layoutAttributesArray?.forEach({ (layoutAttributes) in
+               let itemOffset = layoutAttributes.frame.origin.x
+               if fabsf(Float(itemOffset - horizontalOffset)) < fabsf(Float(offsetAdjustment)) {
```

Comment:
fabsf는 절댓값 비교를 하기 위해 사용하신 걸까요? 
abs를 사용하면 따로 float 변환을 하지 않아도 되니 참고 부탁드립니다:)

```suggestion
              if abs(itemOffset - horizontalOffset) < abs(offsetAdjustment) {
```

Reply:
float 타입으로 비교하기 위해서 사용했습니다 !